pkg/views/workspace/create/summary.go

```diff
@@ -157,8 +152,26 @@ func projectDetailOutput(projectDetailKey ProjectDetail, projectDetailValue stri
 	return fmt.Sprintf("\t%s%-*s%s", lipgloss.NewStyle().Foreground(views.Green).Render(string(projectDetailKey)), DEFAULT_PADDING-len(string(projectDetailKey)), EMPTY_STRING, projectDetailValue)
 }
 
+func calculateViewportSize(content string) (width, height int) {
```

Comment:
<img width="799" alt="Screenshot 2024-10-23 at 10 26 24" src="https://github.com/user-attachments/assets/968b6f58-0dac-4cf6-93e2-b925d07a6dcf">

Why is the height hardcoded to 25? On lower heights the view gets cut off. We should calculate the max height based on the current height of the terminal.

<img width="515" alt="Screenshot 2024-10-23 at 10 26 34" src="https://github.com/user-attachments/assets/251c02f6-efd3-4536-96be-d7c1e8b9f38a">

The same problem applies if the width is to low. Can that be addressed as well?

Reply:
done